demos/drawer/persistent-drawer.html

```diff
@@ -141,6 +141,7 @@ <h1 class="mdc-typography--display1">Persistent Drawer</h1>
         <button type="button" class="mdc-button mdc-button--stroked mdc-button--dense demo-toolbar-example-heading__rtl-toggle-button">Toggle RTL</button>
       </main>
 
+      <script src="/assets/common.js" async></script>
```

Comment:
Nit: I think we'd been standardizing on putting common.js after mcw.js (I realize now I missed this in top-app-bar).